addons/l10n_ec_ats/tablas_model/data/codigo.retencion.csv

```diff
@@ -0,0 +1,122 @@
+id,name,description

```

Comment:
Your file is missing the id of every record... is this the desired behavior?, why?